src/localization/bundles/en.json

```diff
@@ -28,6 +28,8 @@
     "wizard.loading.pleasewait": "Loading... Please wait",
     "logviewer.autoscroll": "Automatically scroll to the bottom",
     "no.options.available": "No options available",
+    "no.component.select": "No Component Selected",
```

Comment:
Per the bug report, when no model is selected, it is supposed to say "No model selected", and with this change it says "No Component Selected"

Reply:
@GarySmith  thanks for the review. Ah..good catch...I just reused whatever it is there. I will make the wording change and also update the commit messages.